scripts/todolist.js

```diff
@@ -0,0 +1,77 @@
+import { add, read, remove, editOneElement, edit } from "./jsonFileStorage.js";
+
+const filename = "public/data.json";
+
+export const toDoListHandler = (action, newItem, editedItem) => {
+  let idx;
```

Comment:
at a glance, this variable doesn't give me much context about what it's suppose to store. maybe you can consider renaming it to something more meaningful